app.py

```diff
@@ -0,0 +1,58 @@
+import dash
+import dash_html_components as html
+import dash_core_components as dcc
+import altair as alt
+import pandas as pd
+from vega_datasets import data
+
+alt.data_transformers.disable_max_rows()
+data = pd.read_csv('/Users/neelphaterpekar/Desktop/MDS/MDS_Block4/DSCI_532/group_6/data/wine_data.csv')
```

Comment:
having the app.py in the root of project and data set in data/raw/wine_data.csv folder should work. I'm using:

data = pd.read_csv('data/raw/wine_data.csv')